core/crypto/murmur/murmur.cpp

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "crypto/murmur/murmur.hpp"
+
+#include <common/buffer.hpp>
+
+namespace fc::crypto::murmur {
+  uint64_t getUint64LE(gsl::span<const uint8_t> bytes) {
+    return bytes[0] | (bytes[1] << 8) | (bytes[2] << 16) | (bytes[3] << 24)
```

Comment:
add static casts.
is it ok, that you are taking into account only little endian?

Reply:
This algorithm works only with LE